steps/github/get-tags/main.go

```diff
@@ -72,21 +78,15 @@ func (gh *GHCommand) Run() (int, []byte, error) {
 	client := githubv4.NewClient(httpClient)
 	err := client.Query(context.Background(), &query, variables)
 	if err != nil {
-		return step.ExitCodeFailure, nil, fmt.Errorf("get-steps failed with error: error: %w", err)
+		return nil, status.FromError(codes.Internal, "get-tags failed with error", err)
 	}
 	tags := []Tag{}
-	output := output{tags}
+	result := output{tags}
 	for _, edge := range query.Repository.Refs.Edges {
 		tag := Tag{edge.Node.Target.Tag.Name, edge.Node.Target.Tag.Message}
-		output.AddItem(tag)
-	}
-
-	ret, err := json.Marshal(output)
-	if err != nil {
-		return step.ExitCodeFailure, nil, fmt.Errorf("get-steps failed with error: error: %w", err)
-
+		result.AddItem(tag)
 	}
-	return step.ExitCodeOK, ret, err
+	return map[string]interface{}{"tags": result.Tags}, status.OK
```

Comment:
Same, check if it works
```suggestion
	return step.Output{"tags": result.Tags}, status.OK
```